### PR TITLE
App Layouts Page - Adds App Container and Page Container diagrams

### PR DESCRIPTION
**Pull request by halocline**
[Deploy Preview](https://deploy-preview-2175--keen-mayer-a86c8b.netlify.app/templates/app-layouts)
[Figma](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Layouts?node-id=1876%3A16251)

#### What does this PR do?

**NOTE:** This PR is to merge into the PAGE-LAYOUTS staging branch.
- Adds the first two diagrams to the App Layouts page, App Container & Page Container.

#### Where should the reviewer start?

[App Layouts Page](https://deploy-preview-2175--keen-mayer-a86c8b.netlify.app/templates/app-layouts)

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [x] Small, medium, and large screen sizes
- [x] Cross-browsers (FireFox, Chrome, and Safari)
- [x] Light & dark modes
- [x] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [x] Console is free of warnings and errors
- [x] Passes E2E commit checks
- [x] Visual snapshots are reasonable

### CODE REVIEW

File: aries-site/src/examples/templates/app-layouts/anatomy/PageAnatomy.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <ContentArea
+      title="Page Container"
+      border={{ color: 'border-weak' }}
+      gap="small"
+      round="xsmall"
+      height={height}
+      width={width}
+    >
+      <ContentArea title="Page Header" background="purple!" flex={false} />
```
Comment (taysea) [inline aries-site/src/examples/templates/app-layouts/anatomy/PageAnatomy.js:20]: We had received feedback in the past that using these strong background colors was a bit too bold, which is why we went for the dashed borders on the current implementation -- wondering if we can find a middle ground

File: aries-site/src/pages/templates/app-layouts.mdx [hunk 2]
```diff
@@ -22,10 +33,17 @@ to be used across all HPE applications. For more information, see
...
+    and the Page Content.'
+  pad="none"
+  previewWidth="min-content"
+>
+  <PageAnatomy />
+</Example>
 
 Jump to:
-- [Container Width](#container-width)
+- [Container Width](#page-container-widths)
```
Comment (taysea) [inline aries-site/src/pages/templates/app-layouts.mdx:46]: This section felt a bit disjointed from the content, left be wondering "why am I jumping there, I am not even sure what Grid is or Breakpoints?"

File: aries-site/src/pages/templates/app-layouts.mdx [hunk 3]
```diff
@@ -44,8 +62,8 @@ The Page Container is responsible for the layout for the main content of a page.
 It contains a page's [Page Header](/components/header#header-for-a-single-page) 
 and its Page Content.
 
-### Container Width
-There are 3 content width options from which to choose, depending on 
+### Page Container Widths
+There are 3 Page Container widths from which to choose, depending on 
 the layout use case: [Wide](#wide-default), [Narrow](#narrow) and [Full](#full). 
 
 In all cases, the minimum width of content is `336px`, after which horizontal
```
Comment (taysea) [inline aries-site/src/pages/templates/app-layouts.mdx:69]: Is this a t-shirt size, or where does this derive from?

Comment (halocline) [reply aries-site/src/pages/templates/app-layouts.mdx:69]: Good question. These are some of the things we want to test out to see how design direction translates into dev implementation. What disconnects are there, if any? What are the friction points? ....

In theory, 336 + 24 + 24 = 384 (medium Box). So I'd like to try to implement a few screens at these breakpoints and see where things work/breakdown. That way we can identify any issues in the translation from theory to implementation, have a conversation and then we can iterate from there.

**Review by taysea — APPROVED** [review]
Implementation looks good -- happy to leave my other comments as separate discussion.

**Pull request merged**